src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/store/PbfMemoryStore.java

```diff
@@ -769,9 +765,6 @@ private boolean nodeHasExplicitOsmTags(final Node node)
         }
         else if (nodeTagSize < AtlasTag.TAGS_FROM_OSM.size())
         {
-            logger.error(
```

Comment:
was this removed because it would never hit? Fine with the removal in general.

Reply:
Removed because it was hit in the case of new node creation, which wasn't padded with the new tags yet, so a lot of junk can possibly appear in the logs.